appengine/gophers/gophers-5/main.go

```diff
@@ -84,31 +84,31 @@ func indexHandler(w http.ResponseWriter, r *http.Request) {
 	// Create a new Firebase App.
 	app, err := firebase.NewApp(ctx, firebaseConfig)
 	if err != nil {
-		params.Notice = "Couldn't authenticate. Try logging in again?"
+		params.Notice = fmt.Sprintf("firebase.NewApp: %v", err)
 		params.Message = message // Preserve their message so they can try again.
 		indexTemplate.Execute(w, params)
 		return
 	}
 	// Create a new authenticator for the app.
 	auth, err := app.Auth(ctx)
 	if err != nil {
-		params.Notice = "Couldn't authenticate. Try logging in again?"
+		params.Notice = fmt.Sprintf("app.Auth: %v", err)
```

Comment:
These errors are shown to the user.
The new text looks only usable for developers.
Maybe they should be logged, instead?

Reply:
Great idea. Done (and updated the title).